### DISCUSSION TITLE
How can i add other modded axes to the config

### DISCUSSION THREAD

**Discussion by SourishGhosh7** [RakambdaOrg/FallingTree#154] [Q&A] [answered]
How can I add a new custom modded axe 
the modded axe do not have a axe tag so how can i add manually

**Comment by Rakambda** [comment, marked as answer]
Depending on if you use the config file or config GUI the format may be a bit different. But basically you have to add the item id of the axe in the tools whitelist (https://github.com/RakSrinaNa/FallingTree/wiki/Settings---Tools)
